cmd/build_test.go

```diff
@@ -292,6 +292,72 @@ func TestBuildPlanWithPruneUnused(t *testing.T) {
 	})
 }
 
+func TestBuildPlanWithRegoEntrypointAnnotations(t *testing.T) {
+
+	tests := []struct {
+		note       string
+		files      map[string]string
+		shouldFail bool
```

Comment:
I think it could be worth expanding the failure case, maybe? i.e.

```suggestion
		shouldFail error // use nil if none expected
```

Reply:
Rolled in the type change into my latest changes. Thanks for the suggestion! :slightly_smiling_face: